swift/ios-shoppe-demo/Resources/Assets.xcassets/Cart.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Cart.png",
```

Comment:
The icon looks a bit skewed.  What about grabbing the cart icon (for iOS) from https://material.io/resources/icons/?search=cart&icon=shopping_cart&style=baseline.  This is the same icon used in the Angular app.

Reply:
sounds good :)